src/features/navigation/RootNavigator/useInitialScreenConfig.tsx

```diff
@@ -18,39 +17,40 @@ export type InitialScreenConfiguration =
   | ScreenConfiguration<'FirstTutorial'>
 
 export function useInitialScreen(): RootScreenNames | undefined {
-  const { isLoggedIn } = useAuthContext()
-
   const [initialScreen, setInitialScreen] = useSafeState<RootScreenNames | undefined>(undefined)
 
   useEffect(() => {
-    getInitialScreen({ isLoggedIn }).then((screen) => {
+    getInitialScreen().then((screen) => {
       setInitialScreen(screen)
       triggerInitialScreenNameAnalytics(screen)
     })
-  }, [isLoggedIn])
+  }, [])
 
   return initialScreen
 }
 
-async function getInitialScreen({ isLoggedIn }: { isLoggedIn: boolean }): Promise<RootScreenNames> {
-  if (isLoggedIn) {
-    try {
-      const user = await api.getnativev1me()
-      const hasSeenEligibleCard = !!(await storage.readObject('has_seen_eligible_card'))
-      if (!hasSeenEligibleCard && user.showEligibleCard) {
-        return 'EighteenBirthday'
-      }
-      if (user.isBeneficiary && user.needsToFillCulturalSurvey) {
-        return 'CulturalSurvey'
-      }
-    } catch {
-      // If we cannot get user's information, we just go to the homepage
-      return homeNavigateConfig.screen
+async function getInitialScreen(): Promise<RootScreenNames> {
+  try {
+    const user = await api.getnativev1me()
+    const hasSeenEligibleCard = !!(await storage.readObject('has_seen_eligible_card'))
+    if (!hasSeenEligibleCard && user.showEligibleCard) {
+      return 'EighteenBirthday'
+    }
+    if (user.isBeneficiary && user.needsToFillCulturalSurvey) {
+      return 'CulturalSurvey'
     }
+  } catch {
+    // If we cannot get the user's information or have an error with the storage,
+    // we do as if the user is not logged in
```

Comment:
la plupart du temps l'access token aura expiré, et donc `await api.getnativev1me()` va throw. Est-ce que ça vaut le coup de faire 1 retry ?

Reply:
le retry est built-in dans notre client non ? mais je vais check